README.md

```diff
@@ -23,10 +23,10 @@ pip3 install -r requirements.txt
 ```
 
 2. Set the environment variable
-To run locally, you will have to download the GCP credentials and set the $GOOGLE_APPLICATION_CREDENTIALS variable to the json file with the credentials. 
+To run locally, you will have to download the GCP credentials and set the $GOOGLE_APPLICATION_CREDENTIALS variable to the json file with the credentials. You can use the same key as for justicia.
 
 ```
-gcloud iam service-accounts keys create --iam-account=test-pseudocone@bbc-connected-data.iam.gserviceaccount.com key.json
+gcloud iam service-accounts keys create --iam-account=justicia@bbc-datalab.iam.gserviceaccount.com key.json
```

Comment:
shall we try to get `pseudocone` here?

Reply:
we can use the same account for both, it has general bucket read/write permissions, not for individual buckets. I realised that a bit late, so I agree that the name justicia isn't the best for this service account. If you think it's important I can change that, but will now put a comment there to clarify